{"id": "batch_req_688aaee29e8081909c00ee8e8de32e21", "custom_id": "task-shared_ref/en/dpo-llama/outputs-shared_ref-en-dpo-llama-normal-021.jsonl-173", "response": {"status_code": 503, "request_id": "068eeea64ef2d7d7b17c84292230c6c3", "body": {"error": {"message": "The server had an error while processing your request. Sorry about that!", "type": "server_error", "param": null, "code": null}}}, "error": null}
{"id": "batch_req_688aaf1fc0808190acce0bb2ed484189", "custom_id": "task-shared_ref/fr/dpo-llama/outputs-shared_ref-fr-dpo-llama-normal-012.jsonl-129", "response": {"status_code": 500, "request_id": "f04931865a81e98917f1f51095f0c28d", "body": {"error": {"message": "An error occurred while processing your request. You can retry your request, or contact us through our help center at help.openai.com if the error persists. Please include the request ID f04931865a81e98917f1f51095f0c28d in your message.", "type": "server_error", "param": null, "code": "server_error"}}}, "error": null}
{"id": "batch_req_688aaf498ce08190afa2e10310a65b2d", "custom_id": "task-shared_ref/ru/dpo-llama/outputs-shared_ref-ru-dpo-llama-normal-012.jsonl-113", "response": {"status_code": 503, "request_id": "8aa49021a5b9077f01405852146d95d1", "body": {"error": {"message": "The server had an error while processing your request. Sorry about that!", "type": "server_error", "param": null, "code": null}}}, "error": null}
{"id": "batch_req_688aafe50f1c81909d910c14ddcd7371", "custom_id": "task-shared_ref/en/dpo-llama/outputs-shared_ref-en-dpo-llama-simple-201.jsonl-47", "response": {"status_code": 503, "request_id": "c33004142ea620be5ef4bcbbecd24a19", "body": {"error": {"message": "The server had an error while processing your request. Sorry about that!", "type": "server_error", "param": null, "code": null}}}, "error": null}
